Sources/jsonfunctions/Expressions/_Expression.swift

```diff
@@ -1,6 +1,8 @@
 //
 // json-functions-swift
 //
+// Copyright (c) 2022 SAP SE or an SAP affiliate company
```

Comment:
Expression was moved from Parser and modified.

Reply:
ok, will change it..